p2p/src/lib.rs

```diff
@@ -14,5 +14,11 @@
     html_logo_url = "https://raw.githubusercontent.com/informalsystems/tendermint-rs/master/img/logo-tendermint-rs_3961x4001.png"
 )]
 
+#[cfg(test)]
+extern crate quickcheck;
+#[cfg(test)]
+#[macro_use(quickcheck)]
+extern crate quickcheck_macros;
```

Comment:
Looks like it's necessary. Otherwise, I get:

```
error: cannot find attribute `quickcheck` in this scope
   --> p2p/src/secret_connection.rs:495:7
    |
495 |     #[quickcheck]
```

Reply:
In the 2018 edition, macros are part of the module system (re-exported from the crate's toplevel), so you can instead do:

```rust
use quickcheck_macros::quickcheck;
```